examples/hpa/README.md

```diff
@@ -1,3 +1,171 @@
+For further instructions, see https://cloud.google.com/stackdriver/docs/managed-prometheus/hpa
+
+# custom-metrics-stackdriver-adapter example instructions
+
+The Custom Metrics Stackdriver Adapter supports querying metrics from
+Managed Service for Prometheus starting with
+[version v0.13.1 of the adapter](https://github.com/GoogleCloudPlatform/k8s-stackdriver/releases/tag/cm-sd-adapter-v0.13.1).
+
+To set up an example HPA configuration using the Custom Metrics Stackdriver
+Adapter, do the following:
+
+1. Set up Managed Service for Prometheus in your cluster.
+2. Install Custom Metrics Stackdriver Adapter in your cluster.
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml
```

Comment:
nit: use a [permalink](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) to specify the commit, or better yet, the tag, of the version of the yaml to install. This avoids us having to refresh the doc later if the state of `master` ever changes (for example if that repository migrates to the `main` naming convention for the base branch).

Reply:
done